src/Components/SmartComponents/EmptyPage/NoAccessPage.js

```diff
@@ -0,0 +1,52 @@
+import { Bullseye, Button, EmptyState, EmptyStateBody, EmptyStateIcon, Page, Title } from '@patternfly/react-core';
+import { LockIcon } from '@patternfly/react-icons';
+import { Main } from '@redhat-cloud-services/frontend-components';
+import React from 'react';
+import propTypes from 'prop-types';
+import { withRouter } from 'react-router-dom';
+import VulnerabilityHeader from '../VulnerabilityHeader/VulnerabilityHeader';
+import { injectIntl } from 'react-intl';
+import messages from '../../../Messages';
+
+const  NoAccessPage = (props) => {
+    const returnToPreviosPage = () => {
+        if (props.history.length > 4) {
+            props.history.go(-3);
+        }
+        else {
+            window.location.href = window.location.origin;
+        }
+
+    };
+
+    return (
+        <Page>
+            <VulnerabilityHeader title={props.intl.formatMessage(messages.vulnerabilitiesHeader)} showTabs={false} />
+            <Main>
+                <Bullseye>
+                    <EmptyState>
+                        <EmptyStateIcon icon={LockIcon} />
+                        <Title headingLevel="h5" size="lg">
+                            {props.intl.formatMessage(messages.hasNoAccess)}
+                        </Title>
+                        <EmptyStateBody>
+                            {props.intl.formatMessage(messages.contactYouOrganization)}
+                        </EmptyStateBody>
+                        <Button variant="primary" onClick= { returnToPreviosPage }>
+                            {props.history.length > 4 ? props.intl.formatMessage(messages.returnPreviousPage)
```

Comment:
why 4?

Reply:
because, I had to detect if the user accessed vulnerability by directly inputting the url or through landing page. When I logged in props.history length 4 was differentiating point.  This is just my solution, it would be cool if there is better approach. (my solution seems weird to me also)

Reply:
hm, that's fragile what would happen if you add `beta` let's say in the URL? 
Also, if you don't have access can you really go to the landing page?